news/2023/2023-11-08-OWC2023-RO16-recap.md

```diff
@@ -0,0 +1,53 @@
+</div>
+
+### ::{ flag=DE }:: Germany vs. ::{ flag=CA }:: Canada
+
+[**View the VOD here!**](https://www.twitch.tv/videos/1968714286?t=00h35m49s)
+
+And no, your eyes are not deceiving you.
+
+This battle between tournament veterans at this early of a stage happened due to Canada's [exceptional circumstances while playing the Qualifiers at TwitchCon](https://twitter.com/xootynator/status/1716179172270166224), causing their unusual 11th seed placement. The spectators were expecting odds as good as a coin flip, well represented by the [osu! World Cup pick'ems](https://pickem.hwc.hr/tournaments/130) getting [nearly even votes](/wiki/shared/news/2023-11-08-OWC2023-RO16-recap.md/pickem.png). The North American squad had the better start going up 3–1 including an important break point on [Mysteka - Jerrybeans](https://osu.ppy.sh/beatmapsets/1860553#osu/3825151). Germany responded quickly on the low approach rate map [jun with Alison - SUNKiSS DROP \~jun side\~](https://osu.ppy.sh/beatmapsets/1498309#osu/3070802), claiming the break point right back. ::{ flag=DE }:: [Criller](https://osu.ppy.sh/users/8116659)'s SS and ::{ flag=DE }:: [Shiox](https://osu.ppy.sh/users/11921197)'s 1x100 score provided much needed confidence.
+
+However, the slug fest of break points continued on a highly contested [Yamamoto Mineko - Presea](https://osu.ppy.sh/beatmapsets/2082024#osu/4360325) pick. Eventually, this turned out to be the crucial point for Canada, and they converted their first match point into the seed upset 5–2 victory against the bronze medallist of 2022.
+
+### ::{ flag=TH }:: Thailand vs. ::{ flag=NZ }:: New Zealand
+
+[**View the VOD here!**](https://www.twitch.tv/videos/1968375051)
+
+The team from Thailand were swept against the United States back in the Round of 32, while New Zealand just barely missed out on advancing in the winners' bracket against Italy. Thus, the confidence level from the Kiwis seemed quite high, and the odds were in their favour.
+
+As it happens, Thailand struggled quite a bit to win their own first two picks by a margin of less than 30k points on each map. However, New Zealand didn't manage to win a single break point while confidently winning their own picks, which consequently led to the tiebreaker. Here, disaster struck for Thailand as ::{ flag=TH }:: [Deppyforce](https://osu.ppy.sh/users/5286213) disconnected during the last map and a restart was not possible anymore. New Zealand took the lead back and increased it — until Deppyforce resurrected and proceeded to full combo the tiebreaker along with ::{ flag=TH }:: [chests](https://osu.ppy.sh/users/14806365) and ::{ flag=NZ }:: [Zoomer](https://osu.ppy.sh/users/6600930).
+
+The Kiwis have been eliminated after two heartbreaking tiebreaker losses whereas Thailand's roster will continue their journey thanks to their huge carry: ::{ flag=TH }:: [Lesperry](https://osu.ppy.sh/users/18092331).
```

Comment:
```suggestion
As it happens, Thailand struggled quite a bit to win their own first two picks by a margin of less than 30k points on each map. However, New Zealand didn't manage to win a single break point while confidently winning their own picks, which consequently led to the tiebreaker. Here, disaster struck for Thailand as ::{ flag=TH }:: [Deppyforce](https://osu.ppy.sh/users/5286213) disconnected during the last map and a restart was not possible anymore. New Zealand took the lead back and increased it — until Deppyforce resurrected and proceeded to full combo the tiebreaker along with ::{ flag=TH }:: [chests](https://osu.ppy.sh/users/14806365) and ::{ flag=NZ }:: [Zoomer](https://osu.ppy.sh/users/6600930).

That combined with ::{ flag=TH }:: [Lesperry](https://osu.ppy.sh/users/18092331)'s otherwise huge carry for the team, the Kiwis have been eliminated after two heartbreaking tiebreaker losses whereas Thailand's roster will continue their journey in the Quarterfinals to face Ukraine next.
```